packages/workflow/src/TelemetryHelpers.ts

```
@@ -129,106 +107,133 @@ export function generateNodesGraph(
 	const nodeNameAndIndex: INodeNameIndex = {};
 	const webhookNodeNames: string[] = [];
 
-	try {
```

removed try/catch that swallowed all errors